flowmachine/flowmachine/core/preflight.py

```
@@ -0,0 +1,99 @@
+import inspect
+from collections import defaultdict
+from functools import wraps
+
+import typing
+
+import networkx as nx
+import structlog
+
+from flowmachine.core.dependency_graph import (
+    calculate_dependency_graph,
```

Ruff

`flowmachine.core.dependency_graph.calculate_dependency_graph` imported but unused

Remove unused import: `flowmachine.core.dependency_graph.calculate_dependency_graph`

(F401)